.github/workflows/CICD.yml

```diff
@@ -34,7 +34,10 @@ jobs:
         GOPATH: /home/runner/work/HewlettPackard/terraform-provider-oneview/go
 
     - name: Go formatting checks
-      run: if [ ! -z "$(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)" ]; then exit 1; fi
+      run: if ! [ -z "$(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)" ]; then 
+                    echo  -e " $(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)";
```

Comment:
what does this new line add to existing behavior..?

Reply:
It checks for gomft errors. If there are any gofmt error, build will exit with error